misc/dbt-materialize/dbt/adapters/materialize/impl.py

```diff
@@ -64,9 +68,37 @@ def parse(cls, raw_index) -> Optional["MaterializeIndexConfig"]:
             )
 
 
+# NOTE(morsapaes): Materialize allows configuring a refresh interval for the
+# materialized view materialization. If no config option is specified, the
+# default is REFRESH ON COMMIT. We add an explicity attribute for the special
+# case of parametrizing the configuration option e.g. in macros.
+@dataclass
+class MaterializeRefreshIntervalConfig(dbtClassMixin):
+    at: Optional[str] = None
+    at_creation: Optional[bool] = False
+    every: Optional[str] = None
+    aligned_to: Optional[str] = None
+    on_commit: Optional[bool] = False
```

Comment:
As noted above, `at` and `every` will have to be lists, because it's allowed to specify them multiple times. (follow-up PR)